packages/cli/src/actions/init.ts

```diff
@@ -5,7 +5,7 @@ import { prompt } from 'inquirer';
 import { sync } from 'command-exists';
 import { constants as CoreConstants } from '@pipcook/pipcook-core';
 
-import { ora, Constants } from '../utils';
+import { Constants, logInfo, logFail, logSuccess } from '../utils';
```

Comment:
I would suggest moving the log related functions to a separate object `logger` to simplify these names.

Reply:
And the function `logSuccess` has not yet used :)